### ISSUE TITLE
Add License

### ISSUE THREAD

**Issue by vipranarayan14** [indic-dict/stardict-sanskrit-kAvya#9]
Hi, sir! I want to know if I can use the mUlam files of Chandas for my web app [vChandas](https://vchandas.netlify.app).

Also, please add a license to the repo, so that I and other developers can know the usage permissions of its data.

**Comment by vvasuki** [comment]
Done. https://github.com/indic-dict/stardict-sanskrit-kAvya/blob/master/LICENSE.md

**Comment by vipranarayan14** [comment]
Thank you, sir.
So, I can use the files in my project and give attribution by linking to the repo, right?

**Comment by vvasuki** [comment]
Sure.

**Comment by vvasuki** [comment]
You might find @shreevatsa 's https://sanskritmetres.appspot.com/ (has link to git repo at the bottom) of interest (in case you didn't already know).

**Comment by vipranarayan14** [comment]
I had already started developing the project before I came upon his tool and the like, on GitHub.
Then, I decided I want to continue my development because:
- I wanted some extra features.
- There are no JS libraries on Chandas. (I think using JS is advantageous as I don't need to maintain a server and the same codebase can be used for making Web and Command-line interfaces.)
- I didn't know python then.
